md/taxonomy.md

```diff
@@ -256,3 +256,4 @@
   * [Manage multiple references in Business Objects](bo-multiple-refs-tutorial.md)
   * [Map authorization rules](custom-authorization-rule-mapping.md)
   * [Use the rich textarea widget in a mail template](rta-mail-template.md)
+  * [How to work with dates and times in forms and BDM](datetimes-management-tutorial.md)
```

Comment:
Usually, for the sake of visibility, we don't write "how to" in the taxonomy
"Manage dates and times in BDM and User Interfaces"